src/extras/declarative-events/index.js

```diff
@@ -38,7 +38,7 @@ module.exports = registerElement('a-event', {
           'Use https://github.com/ngokevin/aframe-event-set-component instead.');
 
         if (targetSelector) {
-          this.targetEls = this.closest('a-scene').querySelectorAll(targetSelector);
+          this.targetEls = this.el.sceneEl.querySelectorAll(targetSelector);
```

Comment:
We can probably leave this one since it's deprecated, just to make sure we don't break it